tests/scripts/run-chain.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
This script should be improved – more info in the comments for package.json

Reply:
done